recipes/perl-atlas-modules/build.sh

```diff
@@ -16,16 +17,19 @@ PERLLIB="${PREFIX}/lib/perl5/${perl_version}/perl_lib"
 mkdir -p $PERLLIB && chmod a+x $PERLLIB
 
 ## install modules from CPAN directly as they are no conda packages for these modules
+
 cpanm -l $PERLLIB MooseX::FollowPBP \
- 					URI::Escape \
- 					URL::Encode \
- 					Config::YAML \
- 					File::Basename \
- 					Bio::MAGETAB \
-					Date::Parse \
-					Test::MockObject \
-					Text::TabularDisplay \
-					Log::Dispatch::File
+                    URI::Escape \
+                    URL::Encode \
+                    Config::YAML \
+                    File::Basename \
+                    Bio::MAGETAB \
+                    Date::Parse \
+                    Test::MockObject \
+                    Text::TabularDisplay \
+                    Log::Dispatch::File \
+                    IO::CaptureOutput \
+                    Class::DBI
 
 mkdir -p ${PREFIX}/etc/conda/activate.d/
 echo "export export PERL5LIB=$PERL5LIB:$atlasprodDir/perl_modules:$PERLLIB/lib/perl5" > ${PREFIX}/etc/conda/activate.d/${PKG_NAME}-${PKG_VERSION}.sh
```

Comment:
```suggestion
echo "export PERL5LIB=$PERL5LIB:$atlasprodDir/perl_modules:$PERLLIB/lib/perl5" > ${PREFIX}/etc/conda/activate.d/${PKG_NAME}-${PKG_VERSION}.sh
```
I wonder if the extra export was avoiding this from being exported or not having any impact.

Reply:
Thanks Pablo, saw this earlier but forgot to correct it